docs/app.js

```diff
@@ -619,6 +630,14 @@ function stripPlain(buf) {
         }
         return stringToBuffer(input.split("\n").slice(8).join("\n"));
     }
+    // VorarlbergNetz
+    // > Vertragskonto;XXXXXXXXXXXX
+    // > Zählpunkt;ATXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    input = decodeUTF16LE(buf);
```

Comment:
The way how the Netzbetreiber differ regarding on that exported data never cease to surprise me 🤯